src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Extensions/EventGridExtensions.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+using Microsoft.Azure.EventGrid.Models;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Text;
+using System.Text.Json;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.Extensions
+{
+	/// <summary>
+	/// 
+	/// </summary>
+	public static class EventGridEventExtensions
+    {
+		private const string SyncTokenPropertyName = "syncToken";
+
+		/// <summary>
+		/// This method uses an EventGridEvent from EventGrid and tries to create a <see cref="PushNotification"/>
+		/// </summary>
+		/// <param name="eventGridEvent"> EventGridEvent from EventGrid</param>
+		/// <param name="pushNotification"> out parameter set up in this method</param>
```

Comment:
>/// <param name="pushNotification"> out parameter set up in this method</param> [](http://example.com/codeflow?start=2&length=79)

Suggested parameter summary for pushNotification:
"If this method returns true, the pushNotification object contains details populated from eventGridEvent. If this method returns false, the pushNotification object is null."

Reply:
This comment still needs to be addressed.